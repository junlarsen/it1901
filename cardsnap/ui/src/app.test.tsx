import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { App } from './app';

describe('root app component', () => {
  it('should be able to render', () => {
    const component = render(<App />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
