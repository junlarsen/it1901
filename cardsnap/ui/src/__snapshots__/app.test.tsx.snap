// Vitest Snapshot v1

exports[`root app component > should be able to render 1`] = `
<DocumentFragment>
  <div>
    <a
      href="/home"
    >
      Home
    </a>
    <a
      href="/decks/aabbcc"
    >
      Deck aabbcc
    </a>
  </div>
</DocumentFragment>
`;
